composer.json

```diff
@@ -11,19 +11,19 @@
     }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.7|~3.1|~4.0",
-        "symfony/finder": "~2.7|~3.1|~4.0",
-        "symfony/console": "~2.7|~3.1|~4.0",
-        "symfony/intl": "~2.7|~3.1|~4.0",
-        "symfony/translation":  "~2.7|~3.1|~4.0",
-        "symfony/twig-bundle": "~2.7|~3.1|~4.0"
+        "symfony/framework-bundle": "~4.4|~5.0",
```

Comment:
You should use `^4.0|^5.0`. 4.0 should be compatible if we define a conflict with twig version like this : `"twig/twig": "<1.41|<2.10"`